### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Tuesday, 13th of December 2022  17:30 CET / 11:30am EDT / 8:30am PDT

### DISCUSSION THREAD

**Discussion by t-lo** [flatcar/Flatcar#914] [Flatcar Office Hours]
## Agenda for the Flatcar Office Hours on Tuesday, 13th of December 2022  17:30 CET / 11:30am EDT / 8:30am PDT

- Jitsi: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording: [here](https://youtu.be/T9fK8mge_Vk)
- Community Calls calendar (all future calls): [Google Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [ical](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)
  
## Welcome
- Brief intro/check-in of the new participants in the Zoom call. Please introduce yourself and share what brings you here today.

### News
- Releases are on pause during the festive days / new year's eve, back in January
### Flatcar on Linode
- @tormath1 
### Share your Flatcar story
- users and devs welcome!

**Comment by tormath1** [comment]
If there is not yet a scheduled spotlight, I can quickly show Flatcar installation on Linode ?
